webapp/src/immersivereader.tsx

```diff
@@ -216,8 +216,12 @@ export function launchImmersiveReader(content: string, tutorialOptions: pxt.tuto
                 core.warningNotification(lf("Immersive Reader cannot be used offline"));
                 break;
             }
+            case "token": {
+                break;
+            }
             default: {
                 core.warningNotification(lf("Immersive Reader could not be launched"));
+                pxt.tickEvent("immersiveReader.error", {error: JSON.stringify(e)});
```

Comment:
```suggestion
                pxt.reportError("immersiveReader.error", JSON.stringify(e));
```

I think? ping @abchatra to make sure as I'm not sure I remember the normal way to post error ticks

Reply:
Both are fine. reportError goes to exceptions table which is harder to use. I am not sure why we are stringify the result code. 

I see ticks like this:

{"target":"arcade","cookie":"false","stage":"","ElectronVersion":"8.3.1","Electron":"1","PxtElectronVersion":"0.7.5","PxtElectronIsProd":"1","PxtTargetVersion":"1.4.33","ChromiumVersion":"80.0.3987.165","PxtCoreVersion":"6.12.14","PxtElectron":"1","NodeVersion":"12.13.0","error":"{\"statusCode\":403}"}
--

{"target":"arcade","cookie":"true","stage":"","error":"{\"statusCode\":0}"}
--

Why not just write the { error : e.statusCode} ?

Reply:
I actually prefer tickEvent

Reply:
some of the errors don't have status code, or a message. I thought it would be good to just have a catch all, but I can change it

Reply:
There should be reportException to log catch all errors.